scripts/constants.py

```diff
@@ -18,13 +18,13 @@
 KUBERNETES_BRANCH = "release-1.12"
 
 # client version for packaging and releasing.
-CLIENT_VERSION = "8.0.0-snapshot"
+CLIENT_VERSION = "8.0.0a1"
 
 # Name of the release package
 PACKAGE_NAME = "kubernetes"
 
 # Stage of development, mainly used in setup.py's classifiers.
-DEVELOPMENT_STATUS = "4 - Beta"
+DEVELOPMENT_STATUS = "3 - Alpha"
```

Comment:
We labelled the snapshot client version on master branch as "4 - Beta". Here in release-8.0 branch we updated it to "3 - Alpha" to generated the 8.0.0a1 client